### PR TITLE
Update MANIFEST.in to fix packaging issue.

### PR DESCRIPTION
**Pull request by Jasha10**

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -4,7 +4,7 @@
     Instructions:
     # Build:
     rm -rf dist/ omegaconf.egg-info/
-    python setup.py sdist bdist_wheel
+    python -m build
```
Comment (Jasha10) [inline setup.py:7]: I copied this from https://hydra.cc/docs/development/release

**Comment by omry** [comment]
1. News fragment.
2. Add some info for how you tested.

File: news/772.bugfix [hunk 2]
```diff
@@ -0,0 +1 @@
+Fix a packaging issue (missing sdist dependency)
```
Comment (omry) [inline news/772.bugfix:1]: OmegaConf got a maintenance issue type? (file suffix)

File: setup.py (hunk 1 above)
Comment (omry) [reply setup.py:7]: This is fine as long as it works correctly. (It took some effort to get it to work for Hydra, but it's a more complicated project).

**Comment by Jasha10** [comment]
Here is the procedure I used to test:
```text
$ conda activate tmp
$ conda install python pip
...
$ pip install build pytest pytest-mock
...
$ rm -rf dist omegaconf.egg-info
$ python -m build
...
Successfully built omegaconf-2.1.1.dev1.tar.gz and omegaconf-2.1.1.dev1-py3-none-any.whl
$ cd dist
$ tar -xzf omegaconf-2.1.1.dev1.tar.gz
$ cd omegaconf-2.1.1.dev1/
$ pip install .
...
Successfully installed PyYAML-5.4.1 antlr4-python3-runtime-4.8 omegaconf-2.1.1.dev1
$ rm tests/test_pydev_resolver_plugin.py
$ pytest ./tests
...
=========================================================== 4693 passed, 6 xfailed in 10.00s ============================================================
```

File: setup.py (hunk 1 above)
Comment (Jasha10) [reply setup.py:7]: On my local machine it is working without issue.

File: news/772.bugfix (hunk 2 above)
Comment (Jasha10) [reply news/772.bugfix:1]: Nope, the suffixes defined in `pyproject.toml` are:
- feature
- bugfix
- api_change
- docs
- misc

Comment (Jasha10) [reply news/772.bugfix:1]: Want me to add maintenance?

Comment (omry) [reply news/772.bugfix:1]: I think we can use misc for this.

**Review by omry — APPROVED** [review]
Looking good. see comment about news fragment suffix.

File: news/772.bugfix (hunk 2 above)
Comment (Jasha10) [reply news/772.bugfix]: Done in https://github.com/omry/omegaconf/pull/776/commits/2f367b45e9248b5c9d00808df0a55a4b37eaff5a.

**Comment by Jasha10** [comment]
I've used `test.pypi.org` to confirm that this diff will resolve issue #772 when the next release is uploaded to pypi.
```text
$ # download tar file from https://test.pypi.org/project/jasha10-omegaconf/#files
$ wget https://test-files.pythonhosted.org/packages/6f/a0/0a62d62e74c1fe0eb117f5f069cf55a7826d8a3a9215e5240c0fcc1295bf/jasha10_omegaconf-2.1.1.dev1.tar.gz
...
$ tar -xzf jasha10_omegaconf-2.1.1.dev1.tar.gz 
$ cd jasha10_omegaconf-2.1.1.dev1/
$ pip install .
...
$ cd ~/omegaconf.git/tests/
$ pip install pytest pytest-mock
...
$ rm test_pydev_resolver_plugin.py
$ pytest
...
============================================================ 4693 passed, 6 xfailed in 8.86s ============================================================
```

**Comment by omry** [comment]
don't forget to cherry pick to master.

**Pull request merged**